frame/fast-unstake/src/lib.rs

```diff
@@ -46,16 +50,29 @@
 //!
 //! If unsuccessful, meaning that the staker was exposed sometime in the last `BondingDuration` eras
 //! they will end up being slashed for the amount of wasted work they have inflicted on the chian.
+//!
+//! ## Details
+//!
+//! See [`pallet`] module for more information.
+//!
+//! ## Examples
+//!
+//! 1. Fast-unstake with multiple participants in the queue.
+#![doc = docify::embed!("./frame/fast-unstake/src/tests.rs", successful_multi_queue)]
+//!
+//! 2. Fast unstake failing because a nominator is exposed.
+#![doc = docify::embed!("./frame/fast-unstake/src/tests.rs", exposed_nominator_cannot_unstake)]
```

Comment:
The quoting of the example code looks a bit off, maybe you can paste it Verbatim @sam0x17 ?  
![image](https://github.com/paritytech/substrate/assets/10380170/6747c4f3-255d-4de7-85e1-0dcd26a7bfb1)

Reply:
@sam0x17 is on it 🚀